src/tools/PICS-generator/XMLPICSValidator.py

```diff
@@ -0,0 +1,122 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import argparse
+import os
+import sys
+
+from rich.console import Console
+
+# Add the path to python_testing folder, in order to be able to import from matter_testing_support
+sys.path.append(os.path.abspath(sys.path[0] + "/../../python_testing"))
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main  # noqa: E402
```

Comment:
we are considering moving modules around so that we can have test packages in pypi eventually. 

Could we have CI validation that these scripts are runnable?

Reply:
I'm totally up for figuring something out here, this is basically just a "workaround" to be able to get the matter_testing_support but not having to have these tools places in the python_testing folder but instead this tools folder, which IMO made more sense 🤷🏻‍♂️